docs/docs/software-architecture.md

```diff
@@ -102,23 +102,16 @@ There are some key requirements and system constraints that have a significant b
 
 ## 6. Deployment
 
-This web application is a system which is hosted in an API. The database will be hosted by ChainSquad once the software is delivered. Atomic transactions are conducted in the backend, so that the client computer does not have to spend much of CPU power. The user login and the orderbook will be executed in the frontend, so the client computer will need a decent level of performance.
+This UI is a system which is hosted in an API. The database will be hosted by ChainSquad once the software is delivered. Atomic transactions are conducted in the backend, so that the client computer does not have to spend much of CPU power.
 
-### 6.1. Deployment Diagram
-
-Figure 4 follows the Clean Architecture principle. The frontend and database depict the outer blue layer. The orderbook and Blockchain gateway function as controllers and hence, begin the green layer. The HTLC blocks below the Blockchain gateway depict the red layer as they are use cases. The ACCS itself depicts the yellow layer at the core of the Clean Architecture.
-
-![](./docs/static/img/Deployment.png)
-Figure 4: Deployment Diagram (Swapchain, 2020) [4]
-
-### 6.2. Technology Stack Description
+### 6.1. Technology Stack Description
 
 All services will be running in docker containers at first and orchestrated by docker-compose for simplicity sake. As Swapchain gets closer to mid-term release, the project team will be moving to a Kubernetes deployment on GCE. [1][2]
 
 Frontend:
 
 - JavaScript, CSS, HTML
-- React.js
+- React.tsx
 - Nginx container
 
 Backend:
```

Comment:
Sorry, not sure about the grey-ish area: microservice are not used anymore, right?

Reply:
hm, good question